src/SDL.c

```diff
@@ -107,6 +107,52 @@ SDL_NORETURN void SDL_ExitProcess(int exitcode)
 #endif
 }
 
+/* App metadata */
+static SDL_PropertiesID SDL_app_metadata_properties = 0;
```

Comment:
Hey, crazy idea... we already have an app global properties. Does it make sense just to implement this as setting the above properties on that object? Then the application can query the properties directly any time they want.

Reply:
Is there a value to this over just setting hints?

Reply:
There's an actual API function that we can point people at and say they should call it.